powerhome/playbook
# [PLAY-784] Converting Toggle kit to Typescript

**What does this PR do?**
Converting Toggle kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-784)

**How to test?**
1. Go to Toggle kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
